automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetes/KubernetesSpec.scala

```diff
@@ -0,0 +1,243 @@
+package org.broadinstitute.dsde.workbench.leonardo
```

Comment:
Can we put this class in the `org.broadinstitute.dsde.workbench.leonardo.apps` package?